### PR TITLE
Update gitian-builder repository's location

### PR DESCRIPTION
**Pull request by alessio**
- github.com/cosmos/gitian-builder has moved to github.com/tendermint/gitian-builder
- Build with go1.13.3

- [ ] Linked to github-issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote tests
- [ ] Updated relevant documentation (`docs/`)
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Reviewed `Files changed` in the github PR explorer

______

For Admin Use:
- Added appropriate labels to PR (ex. wip, ready-for-review, docs)
- Reviewers Assigned
- Squashed all commits, uses message "Merge PR #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

**Comment by alessio** [comment]
Build job testing is ongoing

File: .circleci/config.yml [hunk 1]
```diff
@@ -423,13 +423,7 @@ workflows:
       - upload-coverage:
           requires:
             - test-cover
-      - reproducible-builds:
```
Comment (alexanderbez) [inline .circleci/config.yml]: Why are we no longer doing this only for `master`? Doesn't this job take an extremely long time (as proved by this very PR).

**Review by alexanderbez — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (alessio) [reply .circleci/config.yml]: I enable it unconditionally to be able to test the CI job

**Review by alexanderbez — APPROVED** [review]
ACK

**Pull request merged**